.versionrc.json

```diff
@@ -0,0 +1,49 @@
+{
+    "tagPrefix": "v",
```

Comment:
`    "version": "0.1.24",` where is the `v`?  :)

Reply:
`"version": "0.1.24"` is the package json version; it is without v, and the `v` is added during tag creation.

Reply:
👍